src/vs/code/electron-browser/issue/issueReporterModel.ts

```diff
@@ -66,7 +66,7 @@ export class IssueReporterModel {
 Issue Type: <b>${this.getIssueTypeTitle()}</b>
 
 ${this._data.issueDescription}
-
+${this.getExtensionVersion()}
```

Comment:
SQL CARBON EDIT for all the changes in the vs files?

Reply:
this is actually reverting it to what is was.